mlserver/rest/server.py

```diff
@@ -61,17 +61,24 @@ async def start(self):
         await self._server.serve()
 
     def _get_config(self):
-        kwargs = {
-            "host": self._settings.host,
-            "port": self._settings.http_port,
-            "root_path": self._settings.root_path,
-            "access_log": self._settings.debug,
-        }
+        kwargs = {}
+
+        if self._settings.custom_rest_server_settings:
+            kwargs.update(self._settings.custom_rest_server_settings)
+
+        kwargs.update(
```

Comment:
In this case (same as gRPC and the metrics server), should it be one or the other? As in, either the custom one or the other one? My main concern is future issues trying to reconcile both.

Reply:
In my opinion, explicitly defined settings should override the custom configuration. This allows users to configure extra parameters while they're still using default values for host, http_port, grpc_port, etc.